### PR TITLE
Fix/api methods fix

### PR DESCRIPTION
**Pull request by King-Julian-Pool**

### CODE REVIEW

File: NegoSudApi/Services/BottleService.cs [hunk 1]
```diff
@@ -129,10 +125,108 @@ public class BottleService : IBottleService
...
+                currentBottle.Volume = bottle.Volume;
+                currentBottle.Picture = bottle.Picture;
+                currentBottle.Year_Produced = bottle.Year_Produced;
+                currentBottle.Alcohol_Percentage = bottle.Alcohol_Percentage;
+                currentBottle.Current_Price = bottle.Current_Price;
+
+
+                if (bottle.Producer != null)
+                {
+                    Producer? producer = await _producerService.GetProducerAsync(bottle.Producer.Id, includes: false);
```
Comment (MaximeDan) [inline NegoSudApi/Services/BottleService.cs:147]:
```suggestion
                    Producer? dbProducer = await _producerService.GetProducerAsync(bottle.Producer.Id, includes: false);
```

File: NegoSudApi/Services/Interfaces/IBottleService.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@ public interface IBottleService
     /// Get a Bottle entity from the database by its id, , including or not subobjects and collections
     /// </summary>
     /// <param name="id">The Bottle's id</param>
-    /// <param name="includes">Indicates whether or not subobjects and collections should be loaded</param>
+    /// <param name="includes">Indicates whether or not subobjects and collections should be loaded (true if not specified)</param>
     /// <returns></returns>
```
Comment (MaximeDan) [inline NegoSudApi/Services/Interfaces/IBottleService.cs:13]: Specify what is return ?

File: NegoSudApi/Startup.cs [hunk 3]
```diff
@@ -35,22 +34,13 @@ public void ConfigureServices(IServiceCollection services)
...
-                options.SignIn.RequireConfirmedAccount = false;
-            })
-            .AddEntityFrameworkStores<NegoSudDbContext>();
-        
-        services.AddRazorPages();
     }
 
     public void Configure(IApplicationBuilder app, IWebHostEnvironment env, NegoSudDbContext dbContext)
     {
-        dbContext.Database.EnsureCreated();
```
Comment (MaximeDan) [inline NegoSudApi/Startup.cs:53]: why ?

File: NegoSudApi/Services/BottleService.cs (hunk 1 above)
Comment (MaximeDan) [reply NegoSudApi/Services/BottleService.cs:147]: not sure about the include relation, just a thought about a name a bit more explicit

File: NegoSudApi/Services/Interfaces/IBottleService.cs (hunk 2 above)
Comment (Slupshi) [reply NegoSudApi/Services/Interfaces/IBottleService.cs:13]: resolved

File: NegoSudApi/Startup.cs (hunk 3 above)
Comment (King-Julian-Pool) [reply NegoSudApi/Startup.cs:53]: Not intended (Error from develop merge ? )
-> resolved

File: NegoSudApi/Services/BottleService.cs (hunk 1 above)
Comment (King-Julian-Pool) [reply NegoSudApi/Services/BottleService.cs:147]: Resolved

File: NegoSudApi/Startup.cs [hunk 4]
```diff
@@ -35,17 +34,9 @@ public void ConfigureServices(IServiceCollection services)
         services.AddScoped<IProducerService, ProducerService>();
         services.AddScoped<IRegionService, RegionService>();
 
-        var connectionString = Configuration.GetConnectionString("DefaultNegoSudDbContext")?? throw new InvalidOperationException("Connection string 'DefaultConnection' not found.");;
```
Comment (King-Julian-Pool) [inline NegoSudApi/Startup.cs:38]: Where does it come from ? Not in develop nor the current fix/API_Methods_Fix !

**Pull request merged**